auth/uauth.go

```diff
@@ -48,7 +48,7 @@ var (
 // CreateAuthInfo interface provides necessary data
 type CreateAuthInfo interface {
 	Config() config.Provider
-	Logger() ulog.Log
+	Logger() *zap.Logger
```

Comment:
Meant to post to this comment, but replied to the email instead so it's at the bottom. Copy-paste:

Ya, this is definitely not black-and-white, definitely should hammer out in
person. My primary concern is stability, not ownership. If we can keep
pushing new versions of Fx without **ever** breaking people using our logging
mechanism, then my vote is for whatever mechanism makes that easiest.